modules/apps/fragment/fragment-service/bnd.bnd

```diff
@@ -5,6 +5,6 @@ Import-Package:\
 	com.liferay.fragment.util.comparator,\
 	\
 	*
-Liferay-Require-SchemaVersion: 2.10.2
+Liferay-Require-SchemaVersion: 2.10.3
```

Comment:
@alberttgomes, can you please adjust the commit message to be like: `Update schema version`?

Once your commit is only adjusting the schema version of the module `fragment-service` then there is no need to emphasize it on the commit message, does it make sense to you?

Reply:
Yes, @natocesarrego I will renamed!!